src/main/java/at/hannibal2/skyhanni/config/commands/Commands.kt

```diff
@@ -216,11 +216,13 @@ object Commands {
         ) { FarmingWeightDisplay.lookUpCommand(it) }
         registerCommand(
             "shcopytranslation",
-            "<language code (2 letters)> <messsage to translate>\n" +
-                "Requires the Chat > Translator feature to be enabled.\n" +
-                "Copies the translation for a given message to your clipboard. " +
-                "Language codes are at the end of the translation when you click on a message."
+            "Translates a message in English to another language via clipboard.\n" +
+                "Uses a 2 letter language code that can be found at the end of a translation message."
         ) { Translator.fromEnglish(it) }
+        registerCommand(
+            "shtranslate",
```

Comment:
note this new command in the changelot, best with differneces to old command

Reply:
I didn't do that bc this was an internal command
but ig I should anyway